wmn-data.json

```diff
@@ -5660,7 +5570,7 @@
        },
        {
         "name" : "Twitter",
-        "uri_check" : "https://nitter.net/{account}",
+        "uri_check" : "https://nitter.privacydev.net/{account}",

```

Comment:
I'm seeing that this nitter instance has been rate limited. Wondering if we should just remove nitter.